integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -265,6 +265,13 @@
   //MySQL database constants
   public static final String MYSQL_VERSION = "5.6";
 
+  //OKE constants
+  public static final boolean OKE_CLUSTER = Boolean.parseBoolean(Optional.ofNullable(System.getenv("OKE_CLUSTER"))
+      .orElse("false"));
+  public static final String NFS_SERVER = Optional.ofNullable(System.getenv("NFS_SERVER"))
+      .orElse("");
```

Comment:
Can't we have a suitable default for NFS_SERVER and FSS_DIR ?  We need this when OKE_CLUSTER is set to true.

Reply:
no we can can't , it is created during runtime, since I create MountTarget at the same time as OKE cluster